tests/coverage/reachable/div-zero/reachable_fail/test.rs

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 
```

Comment:
Same here.

Reply:
This one I actually removed because didn't seem to add anything new.